cli/lib/archive.ts

```diff
@@ -24,7 +24,12 @@ export async function createArchive(
 		} );
 
 		archive.pipe( output );
-		archive.directory( path.join( siteFolder, 'wp-content' ), 'wp-content' );
+		archive.directory( path.join( siteFolder, 'wp-content' ), 'wp-content', ( entry: EntryData ) => {
+			if ( entry.name.includes( '.git' ) || entry.name.includes( 'node_modules' ) ) {
+				return false;
+			}
+			return entry;
+		} );
```

Comment:
Let's apply the prettier format. You can run `node node_modules/prettier/bin/prettier.cjs cli/lib/archive.ts --write`.

```suggestion
		archive.directory(
			path.join( siteFolder, 'wp-content' ),
			'wp-content',
			( entry: EntryData ) => {
				if ( entry.name.includes( '.git' ) || entry.name.includes( 'node_modules' ) ) {
					return false;
				}
				return entry;
			}
		);
```